### PR TITLE
if you accept

### PR DESCRIPTION
**Pull request by dabzse**
there is not just php was written, but js and frameworks as well.... but every contributor have missed to mention one of the most important thing: the official documentation....

and the new videos getting obsolated by the time, but the developers are keeping up-to-date their documentations.
and some practice pages, like w3schools, are very great starting and re-starting points....

thank you

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -208,5 +211,21 @@ This section is in under development
...
 
+### W3Schools (to mention some free and awesome contents)
+| Resource Name                                     |Duration| Resource                     |
+|---------------------------------------------------|--------|------------------------------|
+| **W3Schools**: learn, practice: read, do exercise, do quiz | N/A | https://www.w3schools.com/ |
+
+### FastAPI (written in Python)
+| Resource Name                                     |Duration| Resource                     |
+|---------------------------------------------------|--------|------------------------------|
+| Complete code by [@dabzse](http://dabzse.net) | code in Python3.10 | https://github.com/dabzse/FastAPI-presentation |
```
Comment (thecodeholic) [inline README.md:222]: @dabzse This is not even PHP

File: README.md [hunk 2]
```diff
@@ -208,5 +211,21 @@ This section is in under development
...
+
+
+
+### DO NOT FORGET
+| Resource Name                                     |Duration| Resource                     |
+|---------------------------------------------------|--------|------------------------------|
+| "Always" refer to the (official) documentations   |**note**| Official Documentations      |
```
Comment (thecodeholic) [inline README.md:229]: @dabzse That is good advice but not a useful resource for PHP roadmap

File: README.md [hunk 3]
```diff
@@ -162,7 +165,7 @@ This section is in under development
 |------------------------------------------------------------|--------|------------------------------|
 | Laravel E-Commerce                                         | 16h 30m| https://www.youtube.com/watch?v=o5PWIuDTgxg&list=PLEhEHUEU3x5oPTli631ZX9cxl6cU_sDaR |
 | Laravel Movie App                                          | 1h 25m | https://www.youtube.com/watch?v=9OKbmMqsREc&list=PLEhEHUEU3x5pYTjZze3fhYMB4Nl_WOHI4 |
-
+| a lot of great videos from Laraveller                      | collection| https://www.youtube.com/c/Laraveller/videos
```
Comment (thecodeholic) [inline README.md:168]: @dabzse This is fine.

**Review by thecodeholic — COMMENTED** [review]
Thanks for your effort

**Comment by dabzse** [comment]
angular and other js tutorials are still not php.
in the university we made php backend (api with mysql) and c# frontend (with restsharp). as the professor said: everything is connected to everything.
on the internship: laravel backend and angular frontend.

I agree: the FastAPI is absolutely not php, but is a good starting point for creating a frontend around it. (will remove from the list)

if I will remove all my changes except Laraveller, I think you will accept then. am I right?
php manual is really useful. may I add it? there are really good answers.

**Comment by thecodeholic** [comment]
Angular is not PHP you are right. But those are the most popular frontend frameworks and if you want to become a full stack PHP developer you will probably need Angular, React or Vue. That's why I included.
I am fine with links related to PHP backend or PHP full stack developer.
Yes, having Laravel link is fine.
Having official documentation is also fine, but please put useful links not advices.
I hope thinks makes sense.
I appreciate any contribution, but I want this repository to only contain useful links and resources.

**Comment by dabzse** [comment]
- repo renamed to the original one
- removed all changes except:
  - laraveller
  - unneeded `<br>` tags are deleted
  - the list is ordered, and contains the numbers only once

I understand what you need. 
unfortunately I don't have too much time for coding. and to watch videos. but I know, John Elder (Codemycom), Dary Nazar (code with dary), Mosh Hamedani (code with mosh), of course you, and some others with a great videos.
there is Ania Kubów with javascript, but I didn't watch a full tutorial.
for Angular possible great videos are available from Fireship.io (our mentor on internship mentioned them).

one more question: I think only English video tutorials are accepted.

thank you for your time :-)

**Pull request merged**